README.md

```diff
@@ -42,6 +42,19 @@ SPSP_ENDPOINT=https://receiver-endpoint.com npm start
 #### SPSP_ENDPOINT
 * Type: String
 * Description: The receiver's [SPSP endpoint](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/) to which SPSP queries are proxied.
+Mutually exclusive with `SPSP_ENDPOINTS_URL`.
+
+#### SPSP_ENDPOINTS_URL
+* Type: String
+* Description: URL used to fetch a receiver's [SPSP endpoint](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/) to which an SPSP query is proxied.
```

Comment:
Does that mean it can be a payment pointer and a `https://...` url or just the latter?

Reply:
Yes, `SPSP_ENDPOINT` and the returned `spspEndpoint` can be either a payment pointer or url
https://github.com/coilhq/receipt-verifier/pull/19/files#diff-a607ce4168598a55c913ad90b5183266d92cc56f365226c1c32644da6229b283R93